### PR TITLE
Accounts page

### PR DESCRIPTION
**Pull request by moiz-sohail**
Accounts page
Screenshot 2021-11-05 at 7 29 46 PM

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -13,17 +13,18 @@
     "updates": {
```
Comment (kouhaiadan) [inline app.json:13]: these changes are intentional, yes?

File: src/screens/Account.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import Button from "../components/Button";
+import { LoggedOutAccountPlaceholder } from "../image/index";
+
+const Account = () => {
+  return (
+    <SafeAreaView style={tw`bg-white w-full h-full`}>
+      <View style={tw`flex justify-center px-5 h-full`}>
+        <View style={tw`  items-center`}>
+          <Image style={tw`flex`} source={LoggedOutAccountPlaceholder} />
+          <Text style={tw`pl-2 pb-1 pt-4 font-bold text-lg text-black`}>
```
Comment (kouhaiadan) [inline src/screens/Account.tsx:14]:
```suggestion
          <Text style={tw`pl-2 pb-1 pt-4 font-semibold text-lg text-black`}>
```

rest looks good

File: app.json (hunk 1 above)
Comment (moiz-sohail) [reply app.json:13]: these are related with apple sign in

**Review by kouhaiadan — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -9,6 +9,7 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+yarn.lock
```
Comment (faizanaliedu) [inline .gitignore:12]: Reason? We can have this pushed wont effect anything. We do it for web app as well

File: app.json [hunk 4]
```diff
@@ -13,17 +13,18 @@
...
     },
-    "assetBundlePatterns": [
-      "**/*"
-    ],
+    "assetBundlePatterns": ["**/*"],
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "usesAppleSignIn": true,
+      "bundleIdentifier": "com.moizsohail.educativeproject"
```
Comment (faizanaliedu) [inline app.json:20]: Shouldnt be `io.educative.learnerapp`

File: .gitignore (hunk 3 above)
Comment (moiz-sohail) [reply .gitignore:12]: i use yarn on my end instead of npm. To when I am committing it get yarn.lock up in the changed files as well.

**Comment by faizanaliedu** [comment]
Lets resolve conflicts

**Pull request merged**